js/src/wp-seo-wp-global-backport.js

```diff
@@ -0,0 +1,27 @@
+import * as importedData from "@wordpress/data";
+import * as importedElement from "@wordpress/element";
+import get from "lodash/get";
+
+let element = get( window, "wp.element", null );
+let data = get( window, "wp.data", null );
```

Comment:
I think you could put the `importedElement` & `importedData` instead of the null here to skip the later null checks.

Reply:
If you do so, please add documentation above, because that would make what's happening there and why a lot less transparent. Or just don't do it ;)